pyangbind/lib/serialise.py

```diff
@@ -476,7 +476,8 @@ def load_ietf_json(d, parent, yang_base, obj=None, path_helper=None,
 
 class pybindIETFJSONEncoder(pybindJSONEncoder):
   @staticmethod
-  def generate_element(obj, parent_namespace=None, flt=False):
+  def generate_element(obj, parent_namespace=None, flt=False,
```

Comment:
Can you add to the docstring here to explain what `with_defaults` does please? Especially given the comment below where I don't think it matches what I'd expect.